secrets-provider-chart/templates/role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if eq .Values.rbac.serviceAccount.create true -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: {{ .Values.global.namespace }}
```

Comment:
Please use .Release.Namespace instead of .Values.global.namespace in all places.
Then you will not need to pass namespace as a parameter - it will be automatically taken from namespace in which you run helm install